po/LINGUAS

```diff
@@ -7,6 +7,7 @@ he
 it
 ja
 ko
+nb_NO
```

Comment:
Hold your horses bots, this is not enough, it needs to be added into `pkg/shell/manifest.json` as well. Of course this locale does not exist in `po/language_map.txt`

Reply:
Fixed. I want to check preview once it is build as well though.